rust/noosphere/tests/integration.rs

```diff
@@ -99,4 +108,6 @@ async fn single_player_single_device_end_to_end_workflow() {
 
         fs.save(None).await.unwrap();
     };
+
+    drop(temporary_directories)
```

Comment:
Is this needed? My understanding here (`drop` being an [empty function](https://doc.rust-lang.org/core/mem/fn.drop.html) that simply takes ownership) is this is redundant as `temporary_directories` goes out of scope here anyway

Reply:
Yah, IDK what was happening before but I swear it was deleting the directories if I didn't manually drop... took the drops out and it's fine.